src/software-architecture/nodes.md

```diff
@@ -0,0 +1,142 @@
+# Nodes
+
+This section elaborates on all of the ROS nodes that run on your drone to make it fly autonomously. These are described in the order in which they appear in pi.screenrc.
+
+## \`0: rcore
+Starts up a ROS master to allow the nodes to find each other.
+
+* Runs: `roscore`
+
+## \`1: FC
```

Comment:
Why do we have the symbol `\`` in all the headers?

Reply:
It's to indicate that they are e.g. the Screen number 1 and you can access them with \`1. I can clarify it in the text.

Reply:
Ping me here when done

Reply:
@afdaniele I added a clarification.